src/acl.ts

```diff
@@ -94,6 +94,21 @@ export interface AccessControlObject {
   projectTeam: string;
 }
 
+export interface AclMetadata extends BaseMetadata {
+  bucket?: string;
+  domain?: string;
+  entity?: string;
+  entityId?: string;
+  generation?: string;
+  object?: string;
+  projectTeam?: {
+    projectNumber?: string;
+    team?: 'editors' | 'owners' | 'viewers';
+  };
+  role?: 'OWNER' | 'READER' | 'WRITER' | 'FULL_CONTROL';
```

Comment:
is this strict typing in that if new fields are added in the API the node.js client will need to be updated as well?

Reply:
https://github.com/googleapis/nodejs-storage/pull/2234/files#diff-747d768492a54441f1682714c15429c572d74c031145c71672dec819ff94fbbbR109 this would act as a catch all for anything new so that it flows through without us having to make a change. However, we would obviously want to add anything new to the appropriate metadata interface to continue to support up to date typings.

Reply:
I like the `unknown` too - encourages customers to validate it